src/north/CsvToHttp/CsvToHttp.class.js

```diff
@@ -174,12 +267,50 @@ class CsvToHttp extends ApiHandler {
           object.error.push(`Header "${mapping.httpField}": ${response.error}`)
         }
         object.value[field] = response.value
+      } else if (mapping.csvField.match(REGEX_CONTAIN_VARIABLE_STRING)) {
+        // split the input into a array of string
+        // "test ${value}" => [ "test ", ${value}]
+        const csvFieldSplit = mapping.csvField.split(REGEX_SPLIT_TEMPLATE_STRING).filter(Boolean)
+        const field = mapping.httpField
+        csvFieldSplit.forEach((element) => {
+          // match if the string starts with: ${...}
+          if (element.match(REGEX_MATCH_VARIABLE_STRING)) {
+            const headerToGet = element.match(REGEX_GET_VARIABLE)
+            headerToGet.forEach((header) => {
+              const response = CsvToHttp.convertToCorrectType(csvRowInJson[header], mapping.type)
+              if (response.error) {
+                object.error.push(`Header "${mapping.httpField}": ${response.error}`)
+              }
+              if (response.value) {
+                object.value[field] = CsvToHttp.insertValueInObject(object.value[field], response.value)
+              }
+            })
+          } else {
+            object.value[field] = CsvToHttp.insertValueInObject(object.value[field], element)
+          }
+        })
       }
     })
 
     return object
   }
 
+  /**
+   * It returns the concatenation of value with the previous object
+   * If the oject is empty it return the value sent
+   * @param {Mixed} object - object
+   * @param {Mixed} value - value
+   * @return {Mixed} - The converted value
+   */
+  static insertValueInObject(object, value) {
```

Comment:
To make it clearer, I would rename it 'currentJsonValue'. Don't forget to update the jsdoc.

Reply:
And rename value to 'valueToAdd'